AllProtectedFilesUpdate-NDReconnect-1-16m failed for INSUFFICIENT_TX_FEE in precheck

### Description

Same errors happen to AllProtectedFilesUpdate-NIReconnect-1-16m test as well.

This happened in master branch only.

HapiClientValidator has these precheck errors:

```
com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
```



### Steps to reproduce

Slack report link: https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1625342718396700
Results URL: https://console.cloud.google.com/storage/browser/hedera-services-jrs-test-results/hedera-services-automation/master/20210703-194741-GCP-Daily-Reconnect-Correctness-6N-1C/AllProtectedFilesUpdate-NDReconnect-1-16m;tab=objects?authuser=3&prefix=&forceOnObjectsSortingFiltering=false



### Additional context

In hapi_client_combined.log:

```
...
2021-07-03 19:55:48.065 INFO   159  HapiQueryOp - 'UpdateAllProtectedFilesDuringReconnect' - Paying for HapiGetFileContents{sigs=1, payer=GENESIS, node=0.0.3, file=FEE_SCHEDULE, fileId=fileNum: 111
} with body=transactionID { transactionValidStart { seconds: 1625342143 nanos: 1995 } accountID { accountNum: 2 } } nodeAccountID { accountNum: 3 } transactionFee: 100995200 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 2 } amount: -14200 } accountAmounts { accountID { accountNum: 3 } amount: 14200 } } }; sigs=sigPair { pubKeyPrefix: "\n" ed25519: "\f\034x\352\371\3406\246\035b\351\213\204\377\222\016\n\035u\032$\237\337\276\333\'\336\335|\002\005\271\373.@\347\321\335\335\216\213\030\226\366\322qg\037\323B\342BRc\237\202\237\367`\252\256\233(\n" }
com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.submitOp(HapiQueryOp.java:186)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.transactions.file.HapiFileUpdate.lookupInfo(HapiFileUpdate.java:401)
	at com.hedera.services.bdd.spec.transactions.file.HapiFileUpdate.feeFor(HapiFileUpdate.java:383)
	at com.hedera.services.bdd.spec.HapiSpecOperation.finalizedTxn(HapiSpecOperation.java:318)
	at com.hedera.services.bdd.spec.HapiSpecOperation.finalizedTxn(HapiSpecOperation.java:283)
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:145)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:36)
	at com.hedera.services.bdd.spec.utilops.UtilVerbs.lambda$updateLargeFile$12(UtilVerbs.java:561)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.submitOp(CustomSpecAssert.java:61)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:36)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:45)
	at com.hedera.services.bdd.spec.utilops.UtilVerbs.lambda$reduceFeeFor$7(UtilVerbs.java:448)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.submitOp(CustomSpecAssert.java:61)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.HapiApiSpec.exec(HapiApiSpec.java:317)
	at com.hedera.services.bdd.spec.HapiApiSpec.run(HapiApiSpec.java:227)
	at java.base/java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
	at com.hedera.services.bdd.suites.HapiApiSuite.runSync(HapiApiSuite.java:255)
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuite(HapiApiSuite.java:146)
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuiteSync(HapiApiSuite.java:139)
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runSuitesSync$15(SuiteRunner.java:600)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at com.hedera.services.bdd.suites.SuiteRunner.runSuitesSync(SuiteRunner.java:601)
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runTargetCategories$10(SuiteRunner.java:575)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at com.hedera.services.bdd.suites.SuiteRunner.runTargetCategories(SuiteRunner.java:575)
	at com.hedera.services.bdd.suites.SuiteRunner.runCategories(SuiteRunner.java:543)
	at com.hedera.services.bdd.suites.SuiteRunner.main(SuiteRunner.java:464)
2021-07-03 19:55:50.170 INFO   322  HapiApiSpec - 'UpdateAllProtectedFilesDuringReconnect' finished initial execution of CustomSpecAssert
io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
	at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:262)
	at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:243)
	at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:156)
	at com.hederahashgraph.service.proto.java.CryptoServiceGrpc$CryptoServiceBlockingStub.cryptoGetBalance(CryptoServiceGrpc.java:1041)
	at com.hedera.services.bdd.spec.queries.crypto.HapiGetAccountBalance.submitWith(HapiGetAccountBalance.java:203)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.timedSubmitWith(HapiQueryOp.java:201)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.submitOp(HapiQueryOp.java:161)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.HapiApiSpec.exec(HapiApiSpec.java:317)
	at com.hedera.services.bdd.spec.HapiApiSpec.run(HapiApiSpec.java:227)
	at java.base/java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
	at com.hedera.services.bdd.suites.HapiApiSuite.runSync(HapiApiSuite.java:255)
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuite(HapiApiSuite.java:146)
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuiteSync(HapiApiSuite.java:139)
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runSuitesSync$15(SuiteRunner.java:600)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at com.hedera.services.bdd.suites.SuiteRunner.runSuitesSync(SuiteRunner.java:601)
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runTargetCategories$10(SuiteRunner.java:575)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at com.hedera.services.bdd.suites.SuiteRunner.runTargetCategories(SuiteRunner.java:575)
	at com.hedera.services.bdd.suites.SuiteRunner.runCategories(SuiteRunner.java:543)
	at com.hedera.services.bdd.suites.SuiteRunner.main(SuiteRunner.java:464)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /35.229.21.226:50211
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
	at io.netty.channel.unix.Errors.throwConnectException(Errors.java:124)
	at io.netty.channel.unix.Socket.finishConnect(Socket.java:243)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:672)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:649)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:529)
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:465)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:835)
2021-07-03 19:55:50.172 INFO   211  HapiSpecOperation - Node 0.0.8 is unavailable as expected!

...


```

### Hedera network

other

### Version

v0.16.0

### Operating system

Linux

--- a/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java
@@ -206,7 +206,6 @@ public abstract class HapiSpecOperation {
 				updateStateOf(spec);
 			}
 		} catch (Throwable t) {
-			t.printStackTrace();
 			if (unavailableNode && t.getMessage().startsWith("UNAVAILABLE")) {
 				log.info("Node {} is unavailable as expected!", HapiPropertySource.asAccountString(node.get()));
 				return Optional.empty();

